src/main/java/org/jenkinsci/plugins/mesos/OfferQueue.java

```diff
@@ -22,7 +22,7 @@
     private final Logger logger = Logger.getLogger(JenkinsScheduler.class.getName());
 
     private static final int DEFAULT_CAPACITY = 100;
-    private static final Duration DEFAULT_OFFER_WAIT = Duration.ofSeconds(30);
+    private static final Duration DEFAULT_OFFER_WAIT = Duration.ofSeconds(600);
```

Comment:
Why change this value here?

Reply:
Here I just enlarged the race-condition window (as explained in issue #328 ) to make my tests easier and the modified model more convincing.